.travis.yml

```
@@ -0,0 +1,2 @@
+  language: node_js
+  node_js: "current"
```

```suggestion
node_js: "current"
services:
  - mongodb
env:
  - MONGODB_URI=mongodb://localhost:27017/users AUTH_SECRETTEONNAt
```

MONGODB_URI=mongodb://127.0.0.1:27017/auth
AUTH_SECRET=s3cret